src/lib/Scenes/Artwork/Components/ArtworkExtraLinks.tsx

```diff
@@ -91,7 +91,7 @@ export class ArtworkExtraLinks extends React.Component<ArtworkExtraLinksProps> {
           </Sans>
         </>
       )
-    } else if (isAcquireable) {
+    } else if (isAcquireable || isOfferable) {
```

Comment:
This was just an oversight from before-- we should render these FAQs if the work is available for buy now _or_ make offer.